rpc/Cargo.toml

```diff
@@ -30,6 +30,9 @@ tracing = "0.1.37"
 http = "0.2.9"
 jsonrpsee = { version = "0.20", features = ["http-client", "ws-client"] }
 
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+jsonrpsee = { version = "0.20", features = ["wasm-client"] }
+
```

Comment:
Thanks for the context! Will fix and sign.

Reply:
> why are we requiring it behind a (browser-bound) wasm-bindgen feature flag?

The thing is that this particular client will be only enabled when we run in browser. In other wasm environments that supports http/ws there is still a possibility to implement the client manually and still use traits provided by this crate, but if we won't guard it by a feature but rather just a `wasm32` it will fail to compile.

Reply:
I see, that makes sense - thanks!